norse/torch/module/receptive_field.py

```diff
@@ -157,16 +160,22 @@ def __init__(
         scales: torch.Tensor,
         angles: torch.Tensor,
         ratios: torch.Tensor,
+        x: torch.Tensor,
```

Comment:
Would it make sense to default the x, y coordinates to 0? I assume most people want centered RFs. Or?

Reply:
I agree with your comment. I will add the default to 0